src/components/interaction-area/interaction-area.utils.js

```diff
@@ -90,11 +90,18 @@ export const getZoomTransformation = (videoElement, interactionAreaItem) => {
 export const setInteractionAreasContainer = (videojs, newInteractionAreasContainer) => {
   const currentInteractionAreasContainer = getInteractionAreasContainer(videojs);
 
+  newInteractionAreasContainer.style.visibility = 'hidden';
```

Comment:
I'm unfamiliar with what you're trying to do here, but from the description it sounds like hiding the current area is supposed to be the solution no?

Reply:
Ok... so after debug it I found out what causes the glitch .. the image takes some time to get loaded so the title is jumping ... the container gets replace with a new element so there is not state there is more than one in the dom.

As you can see in the picture I gave the image a height to fix the glitch.

![Screen Shot 2021-07-22 at 7 49 44](https://user-images.githubusercontent.com/76470570/126591484-788547e8-af00-4dd4-b29d-e1ea871a82f2.png)

Reply:
great